src/main/starlark/builtins_bzl/common/cc/attrs.bzl

```diff
@@ -119,10 +119,36 @@ most build rules</a>.
    and that rule's <code>outs</code> are automatically added to
    this <code>cc_library</code>'s data files.
 </p>
-<p>Your C++ code can access these data files like so:</p>
+<p>Using the functionality defined by the <code>runfiles_src.h</code> located under 
+   <code>tools/cpp/runfiles/runfiles_src.h</code> Your C++ code can access these data files,
```

Comment:
Drop `_src`, that suffix is only there before packaging.

Please also mention the target under `@bazel_tools` that provides this header.

Reply:
Is this correct?